.github/CODEOWNERS

```diff
@@ -100,7 +100,7 @@ extension/storage/filestorage/                           @open-telemetry/collect
 
 internal/aws/                                            @open-telemetry/collector-contrib-approvers @Aneurysm9 @mxiamxia
 internal/docker/                                         @open-telemetry/collector-contrib-approvers @rmfitzpatrick @jamesmoessis
-
+internal/exp/metrics/                                    @open-telemetry/collector-contrib-approvers @sh0rez
```

Comment:
@RichieSams, how do you feel about being a code owner of this with @sh0rez ?

Reply:
That's fine by me. I already was adding myself in the followup PR: https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/31089

Reply:
@RichieSams, you'll have to become a member of the OpenTelemetry org in order to be a code owner. [Requirements are here](https://github.com/open-telemetry/community/blob/main/community-membership.md#requirements). I'm happy to be one of your sponsors.

Reply:
that would apply to me as well then. @djaglowski would you be willing to be a sponsor for me as well?

Reply:
@jpkrohling Would you be willing to be our second sponsor?

Reply:
Absolutely

Reply:
https://github.com/open-telemetry/community/issues/1945